lab_1_classify_by_unigrams/start.py

```diff
@@ -1,8 +1,9 @@
 """
 Language detection starter
 """
-# pylint:disable=too-many-locals, unused-argument, unused-variable
+from lab_1_classify_by_unigrams import main as func
```

Comment:
why were you imported this like this?

Reply:
because if i just imported like: “from main import “functions_names”” MyPy was shown a mistake “unable to import main” or something like this, so i changed a way of importing to this one (and I import as func, so that there is no confusion with the function “main” in start.py)

Reply:
`blabla` is an example of course

Reply:
done